src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    double getFigureArea();
```

Comment:
```suggestion
    double getArea();
```

as we can use this interface not only for Figure class